_health-care/_js/_form.js

```diff
@@ -6,29 +6,30 @@ function identityTransform(formData, arg) { return arg; }
 function extractTransform(formData, arg) { return formData[arg]; }
 function extractMonthTransform(formData, arg) {
   // Stupid js indexes months from 0.
-  return (new Date(extractTransform(formData, arg))).getMonth() + 1;
+  return extractTransform(formData, arg + "[month]");
 }
 function extractDateTransform(formData, arg) {
-  return (new Date(extractTransform(formData, arg))).getDate();
+  return extractTransform(formData, arg + "[date]");
 }
 function extractYearTransform(formData, arg) {
-  return (new Date(extractTransform(formData, arg))).getFullYear();
+  return extractTransform(formData, arg + "[year]");
 }
 function extractDateWithSlashTransform(formData, arg) {
-  let date = new Date(extractTransform(formData, arg));
   let dateString = "";
 
-  if (date.getMonth() < 10) {
+  let month = extractMonthTransform(formData, arg);
+  let date = extractMonthTransform(formData, arg);
+  if (Number(month) < 10) {
```

Comment:
Should we somehow check if `month` and `date` are blank before modifying `dateString`? Not sure what's better to submit if the field is blank: `''` or `'0/0/'` (`year` isn't being set to 0).